tests/x11/nis_client.pm

```diff
@@ -65,16 +72,18 @@ sub nfs_shares_tab {
     send_key 'alt-a';          # add
     assert_screen 'nis-client-add-nfs-share';
     # On 15-SP2 "Any (Highest Available)" is selected by default, just keep it.
-    if (is_sle('>=15-SP2')) {
-        assert_screen 'nis-client-default-nfs-version';
-    }
-    else {
-        send_key 'alt-v';      # NFSV4 share checkbox (selectable box for newer products)
-        assert_screen 'nis-client-nfsv4-share';
-    }
+    assert_screen 'nis-client-default-nfs-version';
     send_key 'alt-s';          # choose NFS server button
-    assert_screen 'nis-client-nfs-server';
-    send_key 'alt-o';          # OK
+    assert_screen [qw(nis-client-nfs-server no_nfs_server_found)];
+    if (match_has_tag 'no_nfs_server_found') {
+        record_soft_failure "bsc#1167589 - yast2 nis and nfs clients do not find server on the same lan";
+        # Validate then enter address manually
+        send_key 'alt-o';      # OK
+        wait_screen_change { send_key "alt-n" };
+        type_string '10.0.2.1';
+    } else {
+        send_key 'alt-o';      # OK
```

Comment:
same here, to not type anything else unless proper needle is created if screen will change in future, so it will fail in next assert without unexpected actions.